mule-extensions-api/src/main/java/org/mule/runtime/extension/api/runtime/operation/Result.java

```diff
@@ -96,7 +93,7 @@ protected Builder() {}
      * @param length
      * @return
      */
-    public Builder<T, A> length(long length) {
+    public Builder<T, A> length(Long length) {
```

Comment:
Why `Long`, surely if you don't want to set a length, you'd not use this builder method instead of pass null here.  Or do you need to reset length when copying a result or something?